lib/c_glib/src/thrift/c_glib/transport/thrift_platform_socket.h

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
This looks a lot like the platform socket code that already exists in the C++ library.
Given their similarity should they be combined and moved into a common location?